core/drivers/crypto/caam/caam_key.c

```diff
@@ -743,6 +745,12 @@ enum caam_status caam_key_init(void)
 
 	assert(alloc_size <= CFG_CORE_BIGNUM_MAX_BITS);
 
+	key_blob_modifier = caam_calloc_align(KEY_BLOB_MODIFIER_SIZE);
+	if (!key_blob_modifier)
+		return CAAM_FAILURE;
+
+	memcpy(key_blob_modifier, KEY_BLOB_MODIFIER, KEY_BLOB_MODIFIER_SIZE);
```

Comment:
I suggest to add `static_assert(sizeof(KEY_BLOB_MODIFIER) >= KEY_BLOB_MODIFIER_SIZE)` for consistency.
Or, alternatively:`#define KEY_BLOB_MODIFIER_SIZE (sizeof(KEY_BLOB_MODIFIER) - 1)`.

Reply:
Ok.